### PR TITLE
Display envelopes on transaction detail page

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This PR lists secure envelopes on the `TransactionDetail` page. 

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

Note: This image is of the view when there aren't any secure envelopes to display. This will have to be tested with envelopes displayed.

https://tinyurl.com/26yw9olr

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/templates/partials/secure_envelope/secure_envelope_detail.html [hunk 1]
```diff
@@ -1 +1,9 @@
-<div></div>
\ No newline at end of file
+{{ if .SecureEnvelope.Payload }}
```
Comment (daniellemaxwell) [inline pkg/web/templates/partials/secure_envelope/secure_envelope_detail.html:1]: This is a basic implementation for now. The partial will be amended to style the payload and error in a follow on story.

File: pkg/web/templates/partials/secure_envelope/secure_envelope_list.html [hunk 2]
```diff
@@ -1 +1,37 @@
...
+<section>
+  <h2 class="text-xl font-bold">Secure Envelopes</h2>
+  {{ if .SecureEnvelopes }}
+  {{ range .SecureEnvelopes }}
+  <div class="join join-vertical w-full">
+    <div class="collapse collapse-plus join-item border border-base-300">
+      <input type="radio" name="my-accordion-4" checked="checked" />
+      <div class="collapse-title text-xl font-medium">
+        <div class="flex items-center gap-x-1">
+          {{ if eq .Direction "in" }}
```
Comment (daniellemaxwell) [inline pkg/web/templates/partials/secure_envelope/secure_envelope_list.html:10]: These are the same icons displayed on the transaction list page. I'd like to use an alternative, but didn't quite find what I was looking for on FA. I can look at other sites in the future.

File: pkg/web/api/v1/transaction.go [hunk 3]
```diff
@@ -206,6 +208,10 @@ func (c *Transaction) Model() (model *models.Transaction, err error) {
 	return model, nil
 }
 
+func (c *Transaction) TitleStatus() string {
```
Comment (daniellemaxwell) [inline pkg/web/api/v1/transaction.go:211]: The status on the info page looked a bit better in title case, so I created this function. However, if it's preferred to handle this another way, let me know.

**Review by bbengfort — APPROVED** [review]
:shipit: review during Envoy UI meeting.

**Pull request merged**